app/views/entities/index.html.erb

```diff
@@ -0,0 +1,7 @@
+<%= @group.group_name %>
+<% @entities.each do|entity| %>
+    <%= entity.entity_name %>
+<% end %>
+
+<div><%= link_to 'Add a new transaction', new_group_entity_path(@group), class: 'footer-button' %>
+</div>
```

Comment:
- [ ] As per the project [requirements](https://github.com/microverseinc/curriculum-rails/blob/main/capstone/rails_capstone.md#:~:text=a%20new%20category.-,Transactions%20page,-For%20a%20given) This should be your transaction page, so please copy the changes you made inside `groups#index` view and paste it here.